tests/test_server.py

```diff
@@ -51,5 +51,25 @@ def get_events():
         assert r.json
         assert len(r.json) == n_events
 
+def test_get_events_from_start_time(flask_client, bucket, benchmark):
```

Comment:
This just seems to be a copy of the one above except for that it creates more events?
Would make more sense to just re-use that test and instead try the limit parameter not specified and specified with -1, 100 and 1000 in get_events and assert on the expected length

Reply:
I had taken the approach that since the function above was only producing 100 items, there wasn't a way to 'fail' the test case initially since 100 was the original limit. 

If that's still okay, I will make the change to combine the and also check for unspecified, '-1', '100', and '1000'.